mlir/test/python/ir/insertion_point.py

```diff
@@ -3,7 +3,13 @@
 import gc
 import io
 import itertools
+import sys
+
 from mlir.ir import *
+from mlir.dialects._ods_common import _cext
+
+sys.path.append(".")
+_cext.globals.append_dialect_search_prefix("custom_dialect")
```

Comment:
I suppose this piggy-backs on the test logic below that creates an operation with "custom" dialect namespace, that is supposed to somehow trigger the dialect loading? It's not obvious from the test what would happen if loading doesn't happen, and how to fix it, so I'd much rather have a separate test for the loading mechanism itself. We can add `_private` methods to `PyGlobals` class for testing purposes, there is precedent. Such a method can, for example, tell you if a dialect prefix is present in the set of loaded dialects.

Reply:
> It's not obvious from the test what would happen if loading doesn't happen

Yes I couldn't figure out how to "surface" the "failing" behavior discussed above - that without this `custom_dialect.custom.py`, `loadDialectModule` happens over and over. 

> Such a method can, for example, tell you if a dialect prefix is present in the set of loaded dialects.

Okay I'll do that.